app/test/testmerginapi.cpp

```diff
@@ -1590,12 +1701,16 @@ void TestMerginApi::deleteLocalProject( MerginApi *api, const QString &projectNa
   LocalProject project = api->getLocalProject( MerginApi::getFullProjectName( projectNamespace, projectName ) );
   QVERIFY( project.isValid() );
   QVERIFY( project.projectDir.startsWith( api->projectsPath() ) );  // just to make sure we don't delete something wrong (-:
-//  QDir projectDir( project.projectDir );
-//  projectDir.removeRecursively();
 
   api->localProjectsManager().removeLocalProject( project.id() );
 }
 
+void TestMerginApi::deleteLocalDir( const QString &dirPath )
+{
+  QDir dir( dirPath );
+  QVERIFY( dir.removeRecursively() );
```

Comment:
ou, hoping noone will do error during development and try to delete `/` here ... what about deleting the local dir relative to the (hardcoded) temp path ? since we know the temp folder location where we store the test results?

Reply:
Good point!